pyproject.toml

```diff
@@ -21,7 +21,7 @@ dependencies = [
     # dask diagnostics is required to spin up the dashboard for profiling.
     "dask[complete]",
     "deprecated",
-    "hipscat>=0.3.8",
+    "hipscat>=0.3.9",
```

Comment:
Again, presumably the library/repository changes are upcoming.

Reply:
Yes - the repo/pypi release is still named 'hipscat', and I wanted the CI to still work until we can fully rename everything.